README.md

```diff
@@ -1 +1,49 @@
-# Hello World
+# Github Governance
+
+This readme is for building out our github gonvernance.
+
+## Top level SWRCB GitHub Documentation that covers our guiding principals, protocols, management decisions, and policies
+Also, this is open to discussion. 
+
+### Things that might go in this documnet or in this repo:
+1. Data Governanace guiding pricipals and documents
+	1. A code of conduct [possibily this one](https://github.com/skills/.github/blob/main/CODE_OF_CONDUCT.md)
+	2. Open data policy documentation
+		1. CHHS has a [Open Data Handbook](https://chhsdata.github.io/opendatahandbook/) we might considering forking or adopting in all or part. [Their githubpage](https://github.com/chhsdata/opendatahandbook)
+	3. [F(findable) A(accessible) I(interoperable) R(reusable)](https://www.go-fair.org/fair-principles/)
+2. Github account management, repo administration, 
+	1. Default Account permissions for WaterBoard members and outside collaborators
+	2. You can define which individuals or teams own certain types or areas of code in a CODEOWNERS file.
+	3. Other ideas to implement
+		1. Contributing guidlines for internal/external collaborators [possibly this one](https://github.com/skills/.github/blob/main/CONTRIBUTING.md)
```

Comment:
Not sure where it would go, but it might be worth pointing to some suggested coding style guides - e.g., for R a good one might be the [tidyverse style guide](https://style.tidyverse.org/index.html). May also point to related tools for checking and/or reformatting code - e.g., for R the [styler](https://github.com/jimhester/lintr) and [lintr](https://github.com/jimhester/lintr) packages.  This might be related to - or could be included in -  the code review process/guidelines? In general, these could help with consistency and re-usability of code and resources posted to the account. As a side note, that guide also has a section on conventions for commit messages and pull requests [here](https://style.tidyverse.org/gitgithub.html).

Reply:
Somewhat related, it might be worth considering some templates/resources/suggestions for project organization and workflow. This could be something we customize, or adopt from existing resources. I think there are many potential resources - e.g., the [AP Datakit](https://datakit.ap.org/) is a somewhat generic tool (with separate templates for python, R, and generic projects), and the [workflowr package](https://github.com/workflowr/workflowr) is an R-specific resource.

Reply:
I've added coding style language down below in an "other considerations" temp section. Not sure where it fits either. It might be that it fits in a larger code review guidelines/protocol doc as you mention. I agree that consistency and re-usability are the key goals here. 

I've added comits and pr tips alongside workflow suggestions in a github useful link section.